lib/tasks/pipeline_monitor.rake

```diff
@@ -197,7 +197,6 @@ class CheckPipelineRuns
       project_id: bm_project.id,
       user_id: bm_user.id,
       input_files_attributes: input_files_attributes,
-      sample_organism: known_organisms,
```

Comment:
Interestingly, the benchmark sample still adds known_organisms into sample_organism. This information is also added to `sample_notes` (in a different format) so it seemed safe to just remove this.